docs/docs/changelog/changelog.md

```diff
@@ -10,6 +10,29 @@ This page contains an overview of the released versions and highlights the major
 point of view (i.e. focus on features). The complete changelog, including the patch
 versions, can be found on the [GitHub release page](https://github.com/camunda/feel-scala/releases).
 
+# 1.17
```

Comment:
❓ We should mention here the new `is defined()` function behaviour ?

Reply:
No, not here. But I will mention the change in the Camunda update guide. :+1: 

We want to move the FEEL language docs soon into the Camunda 8 docs.